anypoint-mq/4.x/modules/ROOT/pages/anypoint-mq-failover.adoc

```diff
@@ -21,23 +21,68 @@ To use the failover feature, do the following:
 * In Anypoint MQ:
 .. Create or modify a queue and xref:mq::mq-failover.adoc#enable-failover[enable failover on the queue].
 
+
 [[enable-failover-feature]]
 == Enable Failover in Your App
 
 To enable the failover feature in your app, set the `cross.region.failover.feature.enabled` system property to `true`.
 This property is set to `false` by default.
 
+[NOTE]
+When the failover property is enabled, the connector handles processing messages for both the primary and failover queues.
+To prevent interruptions, the app can't publish or consume directly from the fallback queue.
+See <<consume-directly-fallback>> for information.
+
 To set this application property in Runtime Manager:
 
 . Sign in to Anypoint Platform and click *Runtime Manager* in the navigation menu.
 . Deploy a new app or change the settings for a deployed app.
-. On the *Properties* tab, specify values for the property:
+. On the *Properties* tab, specify the value for the property, for example:
 +
-image::mq::mq-set-properties-failover-enabled.png["Properties tab with failover properties specified"]
+image::mq-set-properties-failover-enabled.png["Properties tab with failover system property set to true"]
 
 Alternatively, set the property in *Run Configurations* in Anypoint Studio.
 For more information, see xref:mule-runtime::mule-app-properties-system.adoc[System Properties in Mule Apps].
 
+To disable the failover feature in your app, set the `cross.region.failover.feature.enabled` system property to `false`, for example:
```

Comment:
I think we can remove this.. After the feature is turned off from the UI, customers don't really need to modify this system property in their apps.. they only need to set this property to false when they want to consume messages from the fallback queue directly and we're already documenting that below.